Agreements/Internal/TCNE_TCDAG.txt

```diff
@@ -0,0 +1,26 @@
+COPX:EGLL:*:*:LEDBO:*:London TC NE:London TC DAGGA:15000:*:^PAAVO
```

Comment:
I wonder whether we could simplify all of these agreements with a single 'catch-all'?